### PR TITLE
Move twitter and fb scripts to user page only

### PR DESCRIPTION
**Pull request by Farelion**

### CODE REVIEW

File: src/pages/User/index.js [hunk 1]
```diff
@@ -12,6 +12,7 @@ const User = ({
 }) => (
   <Fragment>
     <Helmet>
+      <script src="/components/PullRequests/share.js" type="text/javascript" />
```
Comment (jenkoian) [inline src/pages/User/index.js:15]: Would be nice to keep this in the footer, would that be possible?

File: public/index.html [hunk 2]
```diff
@@ -74,37 +74,6 @@
...
-
-        t._e = [];
-        t.ready = function(f) {
-          t._e.push(f);
-        };
-        return t;
-      })(document, 'script', 'twitter-wjs');
-    </script>
-    <!-- Facebook -->
     <div id="fb-root"></div>
```
Comment (jenkoian) [inline public/index.html:77]: Would be nice to move this too if poss.

**Comment by Farelion** [comment]
I've made some changes, now fb and twitter scripts that we moved from index.html are inside render of new component called UserShare, that is pulled into user page as last component.
They should show up at the end of body, also moved fb-root as you asked.
Let me know if thats better 😄

**Comment by Farelion** [comment]
Last commit imports UserShare component that i forgot to add oops ;D

File: src/pages/User/components/PullRequests/UserShare.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+
+export default class UserShare extends Component {
```
Comment (jenkoian) [inline src/pages/User/components/PullRequests/UserShare.js:3]: Can we use the function method? To be consistent with other components in the app?

`const UserShare = () => (`

**Comment by Farelion** [comment]
Hey, code is working for me, but i can't pass build check and i have no idea why, i'll keep looking.

**Comment by jenkoian** [comment]
@Farelion you can run `npm run prettier-fix` to fix the styling stuff the build is complaining about

**Comment by Farelion** [comment]
After running `prettier-fix` i still couldn't pass test (while uploading with github desktop), i've made some changes to code and it passed (no idea why tho).

**Comment by jenkoian** [comment]
Thanks @Farelion

**Pull request merged**